.gitignore

```diff
@@ -26,6 +26,8 @@ dist
 
 .idea
 /local/
+.golangci.lenient.yaml
+.golangci.bck.yml
```

Comment:
Question: Why those 2 are ignored? backup can be deleted and you don't generate lenient at all here.